expression/bench_test.go

```diff
@@ -204,6 +204,9 @@ var vecExprBenchCases = map[string][]vecExprBenchCase{
 	ast.Cast: {
```

Comment:
Should we split vecExprBenchCases into multiple maps?
For 2 reasons:
1. Conflict will happen when a PR is ready to merge if we only define one map.
2. This map will be too long to read.

Reply:
Would you have any suggestion about how to split this map？